.github/workflows/deploy.yml

```diff
@@ -52,6 +52,7 @@ jobs:
             --no-progress
             --base "$(pwd)/build"
             --accept 100..399,401,403,429
+            --exclude '.*/status/migration/.*'
```

Comment:
It is odd that we don't have the domain here. Is there a way to fix that?

Reply:
Is that important for the context of this PR though?